### PR TITLE
Adjust pi0 model & organize the file structure

### PR DESCRIPTION
**Pull request by LiuZhihao2022**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by XuS1994 — APPROVED** [review]

File: rlinf/envs/libero/utils.py [hunk 1]
```diff
@@ -321,10 +321,10 @@ def get_libero_wrist_image(
     """
     img = obs["robot0_eye_in_hand_image"]
     img = img[::-1, ::-1]  # IMPORTANT: rotate 180 degrees to match train preprocessing
-    #assert isinstance(resize_size, int) or isinstance(resize_size, tuple)
-    #if isinstance(resize_size, int):
+    # assert isinstance(resize_size, int) or isinstance(resize_size, tuple)
```
Comment (LiuZhihao2022) [inline rlinf/envs/libero/utils.py:324]: It seems resize params is not used here. Should we keep it or remove it?

File: rlinf/models/embodiment/openpi_action_model.py [hunk 2]
```diff
@@ -100,81 +121,103 @@ def input_transform(self,obs: dict, transpose = True):
...
+        outputs["actions"] = outputs["actions"][:, : self.config.action_chunk]
         return outputs
-    
+
     def forward(
-        self, 
-        data: dict[str, Tensor], 
+        self,
+        data: dict[str, Tensor],
+        compute_logprobs: bool = True,
```
Comment (LiuZhihao2022) [inline rlinf/models/embodiment/openpi_action_model.py:178]: Params are not used here:
- compute_logprobs: We leave the rollout state (train and eval mode) in the predict_action_batch function, and we only leave compute logprobs in the forward func. I think we don't need to keep this param as it's not used?
- compute_entropy: In flow matching model, the entropy is not an effective indicator as it depends on a hyperparameter and will not change during training. If OpenVLA needs it, we can keep it for consistency but it's not used.
- compute_values: In fact it should depend on the loss func we used(PPO, GRPO). A separate param may cause potential errors such as when compute_values=True and we use GRPO. Though in the current code it looks well, I suggest delete this parameter

File: rlinf/models/embodiment/openpi_action_model.py [hunk 3]
```diff
@@ -194,31 +237,51 @@ def _process_obs_from_env(self, env_processed_obs):
...
+                    processed_obs[key][sub_key] = sub_value.to(
+                        device=device
+                    ).contiguous()
         return processed_obs
-    
-    def predict_action_batch(self, env_processed_obs, mode: Literal["train", "eval"] = "train") -> Tensor:
-        processed_obs = self._process_obs_from_env(env_processed_obs)
+
+    def predict_action_batch(
+        self, env_obs, mode: Literal["train", "eval"] = "train", compute_values=False
```
Comment (LiuZhihao2022) [inline rlinf/models/embodiment/openpi_action_model.py:275]: Same as the comment above, compute_values should depend on the loss func type

File: rlinf/models/embodiment/openpi_action_model.py [hunk 4]
```diff
@@ -280,43 +357,78 @@ def sample_actions(self, observation: _model.Observation, noise=None, mode="trai
...
+    def sample_mean_var_val(
+        self,
+        x_t,
+        idx,
+        state,
+        prefix_pad_masks,
+        past_key_values,
+        mode,
+        denoise_steps,
+        compute_values=False,
```
Comment (LiuZhihao2022) [inline rlinf/models/embodiment/openpi_action_model.py:438]: Same as the comment above

File: rlinf/models/embodiment/openpi_action_model.py [hunk 5]
```diff
@@ -470,16 +601,29 @@ def get_log_prob_value(
...
             chains_next = chains[torch.arange(bsize), denoise_ind + 1]
-            x_t_mean,x_t_std,value_t = self.sample_mean_var_val(chains_pre,denoise_ind,state,prefix_pad_masks,past_key_values,"compute_logprob",self.config.num_steps)
-            log_probs = self.get_logprob_norm(chains_next,x_t_mean,x_t_std)
+            x_t_mean, x_t_std, value_t = self.sample_mean_var_val(
+                chains_pre,
+                denoise_ind,
+                state,
+                prefix_pad_masks,
+                past_key_values,
+                "compute_logprob",
```
Comment (LiuZhihao2022) [inline rlinf/models/embodiment/openpi_action_model.py]: I think maybe we can delete the "compute_logprob" mode and replace it with "train" mode, as they have the same process. I will commit this change later.

File: rlinf/models/embodiment/openpi_action_model.py [hunk 6]
```diff
@@ -334,38 +446,43 @@ def sample_mean_var_val(self,x_t,idx,state,prefix_pad_masks,past_key_values,mode
             x_t,
             t_input,
         )
-        v_t = self.action_out_proj(suffix_out) # [bs,n_action_steps,max_action_dim]
-        # value prediction 
-        breakpoint()
-        if self.config.adv_method == "ppo":
+        v_t = self.action_out_proj(suffix_out)  # [bs,n_action_steps,max_action_dim]
+        # value prediction
+        if self.config.adv_method == "ppo" and compute_values:
```
Comment (LiuZhihao2022) [inline rlinf/models/embodiment/openpi_action_model.py]: The same. If we have to use this param, we may need to use "assert compute_values is True", otherwise it may return a potential bug that we can run PPO without error but value equals to zero.

File: rlinf/models/__init__.py [hunk 7]
```diff
@@ -28,7 +28,7 @@
 from rlinf.config import torch_dtype_from_precision
 
 
-def get_model_config_and_processor(cfg: DictConfig):
+def get_vla_model_config_and_processor(cfg: DictConfig):
```
Comment (LiuZhihao2022) [inline rlinf/models/__init__.py:31]: Where do we use input_processor if we put the prepare_observations func in the model? It seems only been used in prepare_observations before.
